src/test/java/com/afornalik/ox/board/TestBoard.java

```diff
@@ -2,30 +2,33 @@
 
 import org.testng.annotations.Test;
 
-import java.util.TreeMap;
-
+import static org.assertj.core.api.Assertions.*;
 import static org.testng.Assert.*;
 
 @Test
 public class TestBoard {
 
     private static final int BOUND_FOUR = 4;
     private static final int BOUND_THREE = 3;
+    private static final int INDEX_1 = 1;
+    private static final int MINUS_INDEX = -1;
+    private static final FieldStatus FIELD_STATUS_X = FieldStatus.X;
+    private static final FieldStatus FIELD_STATUS_O = FieldStatus.O;
 
     public void shouldCreateBoard3x3() {
         //when
         Board board = new Board(BOUND_THREE);
 
         //then
-        assertEquals(board.getBound(),BOUND_THREE);
+        assertEquals(board.getBound(), BOUND_THREE);
     }
 
     public void shouldCreateBoard4x4() {
         //when
         Board board = new Board(BOUND_FOUR);
 
         //then
-        assertEquals(board.getBound(),BOUND_FOUR);
+        assertEquals(board.getBound(), BOUND_FOUR);
     }
 
```

Comment:
Hm, youre passing int into constructor, which is assigned to Board field and can be returned by simple getter. There is no logic in this, does this really need to be tested?

Reply:
This was first test. actually it doesn't have sense - deleted.